ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -173,7 +174,26 @@ public void shouldNotEnforceTopicExistanceWhileParsing() throws Exception {
   }
 
   @Test
-  public void shouldCleanUpSchemaRegistry() throws Exception {
+  public void shouldCleanupSchemaAndTopicForStream() throws Exception {
+    ksqlEngine.buildMultipleQueries(
+        "create stream bar with (value_format = 'avro') as select * from test1;"
+        + "create stream foo as select * from test1;",
+        Collections.emptyMap());
+    Schema schema = SchemaBuilder
+        .record("Test").fields()
+        .name("clientHash").type().fixed("MD5").size(16).noDefault()
+        .endRecord();
+    ksqlEngine.getSchemaRegistryClient().register("BAR-value", schema);
+
+    assertThat(schemaRegistryClient.getAllSubjects(), hasItem("BAR-value"));
+    ksqlEngine.terminateQuery(new QueryId("CSAS_BAR_0"), true);
+    ksqlEngine.buildMultipleQueries("DROP STREAM bar DELETE TOPIC;", Collections.emptyMap());
+    assertThat(topicClient.isTopicExists("BAR"), equalTo(false));
```

Comment:
nit: here and elsewhere, you can do `assertTrue` and `assertFalse` instead.

Reply:
Yes, but @dguy likes `assertThat` :)

Reply:
in this case i'd use `assertFalse`, i.e., it is testing a boolean